### PR TITLE
Add Timeline Page

### PR DESCRIPTION
**Pull request by samagragupta**

### CODE REVIEW

File: src/components/EventTimeline.js [hunk 1]
```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import { Timeline, TimelineItem }  from 'vertical-timeline-component-for-react';
+
+const hStyle = {
```
Comment (anshumanv) [inline src/components/EventTimeline.js:4]: Move all styling to sass

**Review by anshumanv — COMMENTED** [review]
Quick deploy link needed.

File: src/components/EventTimeline.js (hunk 1 above)
Comment (samagragupta) [reply src/components/EventTimeline.js:4]: Done

**Comment by samagragupta** [comment]
http://chai-sutta.surge.sh/

**Review by rossoskull — CHANGES_REQUESTED** [review]
* Timeline card body's font size is very small
* Change the date format

**Comment by samagragupta** [comment]
Done the changes @anshumanv @rossoskull

**Comment by rossoskull** [comment]
Provide a preview link

**Comment by anshumanv** [comment]
Update

**Comment by aashutoshrathi** [comment]
Surge krdo franz

**Comment by samagragupta** [comment]
@aashutoshrathi @anshumanv http://alike-competition.surge.sh

<blockquote></blockquote>

**Comment by aashutoshrathi** [comment]
Looks good, please resolve conflicts and update your dev branch

**Review by aashutoshrathi — APPROVED** [review]
Merge after conflicts resolved.

**Comment by samagragupta** [comment]
@aashutoshrathi Done

**Comment by aashutoshrathi** [comment]
@samagragupta one more surge link

**Comment by anshumanv** [comment]
😕

**Comment by anshumanv** [comment]
Code quality badha thoda

**Pull request merged**